setup.py

```diff
@@ -32,24 +32,18 @@
     "grpcio >= 1.38.1, < 2.0dev",  # https://github.com/googleapis/python-bigquery/issues/695
     "google-api-core[grpc] >= 1.29.0, < 3.0.0dev",
     "proto-plus >= 1.10.0",
+    "google-cloud-bigquery-storage >= 2.0.0, <3.0.0dev",
```

Comment:
This should still allow 1.x, as per https://github.com/googleapis/python-bigquery/pull/784/files

It also means we probably need to add the minimum version check back. Or wait for https://github.com/googleapis/google-cloud-python/issues/10566 to be resolved.